### PR TITLE
change base image for fluentd-es

### PR DESCRIPTION
**Pull request by andriisoldatenko**

### CODE REVIEW

File: docker/vendor/fluentd/Gemfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+gem 'activesupport', '~>5.2.3'
+gem 'fluentd', '<=1.5.1'
+gem 'fluent-plugin-concat', '~>2.3.0'
+gem 'fluent-plugin-detect-exceptions', '~>0.0.12'
+gem 'fluent-plugin-elasticsearch', '~>3.5.2'
+gem 'fluent-plugin-kubernetes_metadata_filter', '~>2.2.0'
+gem 'fluent-plugin-multi-format-parser', '~>1.0.0'
+gem 'fluent-plugin-prometheus', '~>1.4.0'
+gem 'fluent-plugin-systemd', '~>1.0.2'
+gem 'oj', '~>3.7.12'
```
Comment (andriisoldatenko) [inline docker/vendor/fluentd/Gemfile]: we can move `RUN gem install fluent-plugin-rewrite-tag-filter` to Gemfile i think.

File: docker/vendor/fluentd/Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY:	build push
+
+PREFIX = quay.io/fluentd_elasticsearch
```
Comment (ashb) [inline docker/vendor/fluentd/Makefile]: We can't push to this can we?

File: docker/vendor/fluentd/clean-apt [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# Copyright 2016 Astronomer Inc.
```
Comment (ashb) [inline docker/vendor/fluentd/clean-apt]: Given this was taken from kube protect we should maintain their copy right I think - especially when we don't change the file.

Or at least add an attribution/source in a comment

File: docker/vendor/fluentd/Makefile [hunk 4]
```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2016 Astronomer Inc.
```
Comment (ashb) [inline docker/vendor/fluentd/Makefile]: 2019 now ;)

File: docker/vendor/fluentd/clean-apt (hunk 3 above)
Comment (ashb) [reply docker/vendor/fluentd/clean-apt]: Probably keep the original copyright line and add ours below

File: docker/vendor/fluentd/Makefile (hunk 2 above)
Comment (andriisoldatenko) [reply docker/vendor/fluentd/Makefile]: i think i'll remove it.

**Comment by andriisoldatenko** [comment]
```
Step 13/13 : LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 ---> Running in ff0e9a864ddd
Removing intermediate container ff0e9a864ddd
 ---> 8769d58bbc98
Successfully built 8769d58bbc98
Successfully tagged fluentd-as:latest
```

**Comment by andriisoldatenko** [comment]
@schnie FYI
```
  helm.astronomer.io git:(master) ✗ kubectl get pods --namespace andrii-dev | grep flu
kindred-lizard-fluentd-27tdv                             1/1       Running             0          56s
kindred-lizard-fluentd-2qc76                             1/1       Running             0          56s
kindred-lizard-fluentd-6pxzs                             1/1       Running             0          42s
kindred-lizard-fluentd-6zv27                             1/1       Running             0          56s
kindred-lizard-fluentd-7785x                             1/1       Running             0          56s
kindred-lizard-fluentd-7c9fl                             1/1       Running             0          56s
kindred-lizard-fluentd-7xnqb                             1/1       Running             0          44s
kindred-lizard-fluentd-dbx44                             1/1       Running             0          56s
kindred-lizard-fluentd-dql2l                             1/1       Running             0          56s
kindred-lizard-fluentd-h44vz                             1/1       Running             0          56s
kindred-lizard-fluentd-hhqs5                             1/1       Running             0          56s
kindred-lizard-fluentd-hsfn5                             1/1       Running             0          56s
kindred-lizard-fluentd-mgrrj                             1/1       Running             0          56s
kindred-lizard-fluentd-qctnw                             1/1       Running             0          42s
kindred-lizard-fluentd-r2p2l                             1/1       Running             0          56s
kindred-lizard-fluentd-sqdnn                             1/1       Running             0          44s
kindred-lizard-fluentd-t7lh7                             1/1       Running             0          56s
kindred-lizard-fluentd-zl4t9                             1/1       Running             0          56s

```

File: docker/vendor/fluentd/Dockerfile [hunk 5]
```diff
@@ -13,11 +13,57 @@
...
+                     ruby-dev \
+                     libffi-dev \
+                     ca-certificates \
+                     libjemalloc2 \
+                     ruby \
+    && echo 'gem: --no-document' >> /etc/gemrc \
+    && gem install --file Gemfile \
+    && apt-get purge -y --auto-remove \
+                     -o APT::AutoRemove::RecommendsImportant=false \
+                     $BUILD_DEPS \
```
Comment (ashb) [inline docker/vendor/fluentd/Dockerfile]: The base layer had:

```
RUN BUILD_DEPS="make gcc g++ libc6-dev ruby-dev libffi-dev" \
```

but we don't set that, so those build deps are never removed.

**Pull request merged**